cmd/registry/controller/parser.go

```diff
@@ -61,6 +61,8 @@ func validateGeneratedResourceEntry(parent string, generatedResource *rpc.Genera
 			return resourceName.Version() != ""
 		case "spec":
 			return resourceName.Spec() != ""
+		case "specrev":
```

Comment:
We are not supporting `$resource.specrev` for now. 

I think if we want to support revisions in some way, we can maybe support `$resource.spec@latest` instead of `specrev`, because which revision to pick will again be a question. This can be a discussion outside the scope of this PR though.